recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "array-api-strict" %}
-{% set version = "1.1" %}
+{% set version = "2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/array_api_strict-{{ version }}.tar.gz
-  sha256: 8172ff23d1865d4129b16e3da1d6846bf33474c996cc252bbb13305cdf6ede59
+  sha256: 2f36091d6a5931bbe75db42374d278097cf7d7f3402a55620eccb6a0e3f9f723
 
 build:
   noarch: python
```

Comment:
Should we build it as arch-specific?

Reply:
It's a test dependency. I find it more convenient to keep it as noarch.